client/src/components/Landing.js

```diff
@@ -1,13 +1,31 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import './Landing.css';
 
 class Landing extends Component {
     render() {
         return(
             <div>
-                This is the page you see when you are at our landing page.
+                <div id="background">
+                    <img src={require('../assets/images/landing-temp.jpg')} alt="" />  
+                
+                    <div id="now-playing">
+                        <div class="now-playing-text">
+                            <div id="main-text">You're listening to</div>
+                            <img src={require('../assets/images/album-temp.jpg')} alt="" />
+                            <div class="song-info"> 
```

Comment:
Do we want alternative pics of these don't load for some reason?

Reply:
Yep. I'm planning on creating javascript for it later

Reply:
"alt" is just text actually, not a picture. EX: "Picture not found"